src/templates/main/mixins/user_card_mixin.html

```diff
@@ -27,14 +27,18 @@ <h4><a class="black-text" href="{{ userprofile.get_absolute_url }}">{{ userprofi
             <h4>About {{ userprofile.user.first_name }}</h4>
             <hr>
             <div class="about">
-                <h5><a href="https://api.whatsapp.com/send?phone=91{{ userprofile.phone }}">
-                    <i class="icons-sm-i fa fa-whatsapp green-text"></i></a>
-                    <a href="tel:{{ userprofile.phone }}">+91-{{ userprofile.phone }}</a>
-                </h5>
-                <h6 class="h6-responsive">
-                    <a class="icons-sm email-ic" href="mailto:{{ userprofile.user.email }}">
-                    <i class="icons-sm-i fa fa-envelope-o"></i> {{ userprofile.user.email }}</a>
-                </h6>
+                {% if userprofile.phone %}
```

Comment:
why have you added this

Reply:
otherwise only +91- and email icon only will shown in faculty profile